packages/go/stbernard/command/dora/report.go

```
@@ -0,0 +1,309 @@
+//   - Months: "3m", "6mo", "6months" (assumes 30 days/month)
+//   - Years: "1y", "3yr", "3years" (assumes 365 days/year)
+// Falls back to 90 days if parsing fails.
+func parseDefaultPeriod(period string) int {
+	period = strings.TrimSpace(period)
+	if period == "" {
+		return 90
+	}
+
+	// Detect suffix and multiplier
+	var (
+		multiplier = 1 // Default: days
```

golangci-lint (2.12.2)

[major] ineffectual assignment to multiplier

(ineffassign)